lib/view-models/search-view-model.coffee

```diff
@@ -30,5 +30,5 @@ class SearchViewModel extends ViewModel
       @restoreHistory(@historyIndex)
 
   confirm: (view) =>
-    @vimState.pushSearchHistory(@)
```

Comment:
Yes, I think so: `//<enter>`, `??<enter>`, `/<enter>` and `?<enter>` would all be easy to implement with this, I think. 
I'll be happy to try and add that if the powers that be (@maxbrunsfeld?) like this PR.